src/avk.cpp

```diff
@@ -2314,17 +2314,29 @@ namespace avk
 		
 		return result;
 	}
-	
+
 	std::optional<command_buffer> buffer_t::fill(const void* aDataPtr, size_t aMetaDataIndex, sync aSyncHandler)
 	{
 		auto metaData = meta_at_index<buffer_meta>(aMetaDataIndex);
 		auto bufferSize = static_cast<vk::DeviceSize>(metaData.total_size());
+		return fill_partially(aDataPtr, bufferSize, 0u, aMetaDataIndex, std::move(aSyncHandler));
+	}
+
+	std::optional<command_buffer> buffer_t::fill_partially(const void* aDataPtr, size_t aDataSizeInBytes, size_t aOffset, size_t aMetaDataIndex, sync aSyncHandler)

```

Comment:
```suggestion
	std::optional<command_buffer> buffer_t::fill_partially(const void* aDataPtr, size_t aMetaDataIndex, size_t aOffse, size_t aDataSizeInBytest, sync aSyncHandler)

```
I would like to have a different order of parameters:
- `aMetaDataIndex` at the same spot as in the original `fill`
- `aOffset` before `aDataSizeInBytes`

And, I think, I'd prefer to have this method called `fill` --- i.e. an overload to the original `fill` as you have suggested.